circle.yml

```
@@ -52,7 +52,7 @@ commands:
 jobs:
     check-factory-versions:
         machine:
-            image: ubuntu-2204:2023.10.1
+            image: ubuntu-2204:2024.01.1
```

Are we able to leverage the newest version of the image available here? 
```suggestion
            image: ubuntu-2204:2024.05.1
```

---

@AtofStryker 

I'm proposing it to it step by step based on getting into trouble trying to leap to the latest image in one go. I don't see any disadvantage for users if it takes a bit longer. It just means a couple more PRs before we get to the final machine image version. Each step will get tested out though, giving a bit more confidence on the way.

Thanks in any case for your question / suggestion!

---

I would propose to jump to `2024.05.1` in the next PR.  The intervening images can be skipped. That would be sometime next week:

| Image Tag | Node.js | Docker Engine | buildx  | PR       |
| --------- | ------- | ------------- | ------- | -------- |
| 2024.01.1 | 20.10.0 | 24.0.7        | v0.12.1 | PR #1160 |
| 2024.01.2 | 20.10.0 | 24.0.7        | v0.12.1 | skip     |
| 2024.04.4 | 20.12.2 | 26.0.2        | v0.14.0 | skip     |
| 2024.05.1 | 20.12.2 | 26.0.2        | v0.14.0 | TBD      |